### ISSUE TITLE
`deepForEachFn` does not accept `children` argument as mentioned in the docs.

### ISSUE THREAD

**Issue by bar-amir** [fernandopasik/react-children-utilities#33]
# Bug Report
`deepForEachFn` does not match the docs as it does not accept a `children` argument.

The docs say:
```typescript
deepForEach(
  children: ReactNode | ReactNode[],
  forEachFn: (child: ReactNode, index?: number, children?: ReactNode[]): void,
): void
```

However, in `deepForEach.d.ts`:
```typescript
declare const deepForEach: (children: ReadonlyDeep<ReactNode | ReactNode[]>, deepForEachFn: (child: ReadonlyDeep<ReactNode>, index?: number | undefined) => void) => void;
```

**Comment by fernandopasik** [comment]
Is this erroring when you build typescript? Could you share the code example you are using please ?

**Comment by fernandopasik** [comment]
Fixed in v2.7.1
